README.md

```diff
@@ -18,9 +18,18 @@ npm install eslint-teamcity --save-dev
 ```
 
 ## Usage
+* as onrun eslint formatter:
 ```sh
   eslint --format './node_modules/eslint-teamcity/index.js' myfiletolint.js
 ```
+* as post runnig formatter, work with standart eslint json report
+```sh
+  node index.js result.json
```

Comment:
Should this not be `node_modules/eslint-teamcity/index.js` instead of just `index.js`? Chances are that people won't be running this while inside the eslint-teamcity directory.